packages/design-system/src/components/contextMenu/menuItem.js

```diff
@@ -71,7 +71,7 @@ export const MenuItem = ({
   const handleClick = useCallback(
     (ev) => {
       onClick(ev);
-      onDismiss();
+      onDismiss(ev);
```

Comment:
💭 Thought it was strange that onDismiss wasn't exposing the event here

Reply:
makes sense. just to keep the event handler spec consistent between this and `onClick` . also can't see passing `ev` through here hurting anything. lgtm